src/main/java/org/apache/sling/scripting/jsp/JspScriptEngineFactory.java

```diff
@@ -323,12 +324,12 @@ private void callJsp(final Bindings bindings,
         }
         final SlingIOProvider io = this.ioProvider;
         final JspFactoryHandler jspfh = this.jspFactoryHandler;
-
         // abort if JSP Support is shut down concurrently (SLING-2704)
         if (io == null || jspfh == null) {
             logger.warn("callJsp: JSP Script Engine seems to be shut down concurrently; not calling {}",
                     scriptHelper.getScript().getScriptResource().getPath());
-            return;
+            throw new Exception("callJsp: JSP Script Engine seems to be shut down concurrently; not calling {"+
+                    scriptHelper.getScript().getScriptResource().getPath()+"}");
```

Comment:
You don't need { and } here anymore, the placeholder were replaced by the logging engine in the previous form.